lib/realtime/operations.ex

```diff
@@ -98,4 +98,58 @@ defmodule Realtime.Operations do
       # credo:disable-for-next-line
       IO.inspect({"Failed to stop", tenant, kind, reason})
   end
+
+  def dump_processes do
+    {:ok, device} = File.open("/tmp/erlang_process_dump.csv", [:write, :utf8])
+
+    device
+    |> IO.write(
+      "intial_call\tstatus\tmemory\tstack_size\tcurrent_stacktrace\tmessage_queue_len\tmessages\n"
```

Comment:
You can do `term_to_binary` and just write the whole term from `Process.list()`

Reply:
👍 will change